lib/charms/mongodb/v0/config_server_interface.py

```diff
@@ -0,0 +1,150 @@
+# Copyright 2023 Canonical Ltd.
```

Comment:
When I did `charmcraft create-lib` it created this lib in `v0` ... ideally all the sharding stuff would be in `v1`. What I can do is after this lib is published I could bump the LIPAPI, but that feels messy...

Reply:
I don't think that there is a way around it, most likely chamrcraft will complain about library version.